src/Views/Home.jsx

```diff
@@ -53,6 +53,16 @@ export const Home = () => {
                 </p>
               </Link>
             </div>
+            <div className="tile is-parent">
+              <Link
+                className="pagetile tile is-child has-background-white"
+                to="/resources"
+                onClick={handleNavigate}
+              >
+                <h3 className="pagetitle">Resource</h3>
+                <p className="pagedescriptor">Checkout the resources.</p>
```

Comment:
To match what we have on the channel description on the server itself, this could potentially be something like: *A set of helpful coding resources, compiled by server staff.*